docs/ai/quickstarts/quickstart-openai-generate-images.md

```diff
@@ -26,6 +27,54 @@ Get started with the .NET Azure OpenAI SDK by creating a simple .NET 8 console c
 
     If you get an error message the Azure OpenAI resources may not have finished deploying. Wait a couple of minutes and try again.
 
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="semantic-kernel"
+<!-- markdownlint-enable MD044 -->
+
+## Understanding the code
+
+Our application uses the `Microsoft.SemanticKernel` client SDK, which is available on [NuGet](https://www.nuget.org/packages/Microsoft.SemanticKernel), to send and receive requests to an Azure OpenAI service deployed in Azure.
+
+The `AzureOpenAITextToImageService` service facilitates the requests and responses.
+
+```csharp
+AzureOpenAITextToImageService textToImageService = new(deployment, endpoint, key, null);
+```
+
+The entire application is contained within the _Program.cs_ file. The first several lines of code load secrets and configuration values that were set in the `dotnet user-secrets` for you during the application provisioning.
+
+```csharp
+// == Retrieve the local secrets saved during the Azure deployment ==========
+var config = new ConfigurationBuilder()
+    .AddUserSecrets<Program>()
+    .Build();
+
+var config = new ConfigurationBuilder().AddUserSecrets<Program>().Build();
+string endpoint = config["AZURE_OPENAI_ENDPOINT"];
+string deployment = config["AZURE_OPENAI_GPT_NAME"];
+string key = config["AZURE_OPENAI_KEY"];
+```
```

Comment:
This looks out of order, it should be before the `AzureOpenAITextToImageService` is created shouldn't it?

Reply:
I agree - I was copying the Azure Open AI SDK pattern/ordering but maybe we should change both?

Reply:
@FBoucher  thoughts?

Reply:
Ah, I didn't realise that was how the AOAI SDK was structured.

Reply:
Sounds good  - I'll tackle this in a follow up PR.

Reply:
https://github.com/dotnet/docs/issues/40218